addons/sale/models/sale_order_line.py

```diff
@@ -1185,3 +1185,6 @@ def _is_delivery(self):
     def _is_not_sellable_line(self):
         # True if the line is a computed line (reward, delivery, ...) that user cannot add manually
         return False
+
+    def _sellable_lines_domain(self):
+        return ["('is_downpayment', '=', False)"]
```

Comment:
Do you really need a list of strings? Can't it be a clean domain (not stringified) ?